source/dem/pp_contact_force.cc

```diff
@@ -104,13 +103,8 @@ PPContactForce<dim>::update_contact_information(
   // which were already in contact (pairs_in_contact) needs to
   // modified using its history, while the tangential_overlaps of
   // new particles are equal to zero
-  Tensor<1, dim> modified_tangential_overlap =
-    contact_info.tangential_overlap +
+  contact_info.tangential_overlap +=
     contact_info.tangential_relative_velocity * dt;
-
-  // Updating the contact_info container based on the new calculated values
-  contact_info.tangential_overlap           = modified_tangential_overlap;
-  contact_info.tangential_relative_velocity = tangential_relative_velocity;
```

Comment:
I don't understand what changes have been done here? The tangential overlap is not corrected anymore?

Reply:
It is corrected. The only change that I have made is that before we used tangential_relative_velocity (unundated tangential relative velocity at the current iteration) for updating tangential overlap. Now we use contact_info.tangential_relative_velocity (which is updated at the current iteration), and unnecessary vectors (modified_tangential_overlap and normal_relative_velocity) are removed because they were being used only once in updating the tangential overlap.